packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -573,6 +574,11 @@ export function createAPIFactory(
             },
             get tabGroups(): theia.TabGroups {
                 return tabsExt.tabGroups;
+            },
+
+            // ExternalUriOpener @stubbed
+            registerExternalUriOpener(id: string, opener: theia.ExternalUriOpener, metadata: theia.ExternalUriOpenerMetadata): theia.Disposable {
+                return new Disposable(() => { });
```

Comment:
minor - we can use the following instead:

```suggestion
                return Disposable.NULL;
```